### PR TITLE
Japanese translation

### PR DESCRIPTION
**Pull request by zukizukizuki**
## Overview
The following is a translation of the Japanese text.
https://github.com/DesModder/DesModder/blob/main/docs/DEVELOPMENT.md#example-creating-a-new-translation-file

I have translated the items listed in [en.ftl](https://github.com/DesModder/DesModder/blob/main/localization/en.ftl).
I do not translate content that, in my judgment, can be understood as is.

## issues
https://github.com/DesModder/DesModder/issues/683

### CODE REVIEW

**Comment by zukizukizuki** [comment]
@radian628
Please check it.

File: localization/i18n-core.ts [hunk 1]
```diff
@@ -52,4 +53,5 @@ function addLanguage(locale: string, ftl: string) {
 addLanguage("en", enFTL);
 addLanguage("es", esFTL);
 addLanguage("fr", frFTL);
+addLanguage("fr", jaFTL);
```
Comment (FenyLabs) [inline localization/i18n-core.ts:56]: You probably want addLanguage("ja")

Comment (zukizukizuki) [reply localization/i18n-core.ts]: sorry🙇‍♂️ Im fixed it.

File: localization/ja.ftl [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+## General
+menu-learn-more = 詳細
+menu-desmodder-plugins = DesModderプラグイン
+menu-desmodder-tooltip = DesModderメニュー
+
+## Category names
+category-core-name = カテゴリ名
+category-utility-name = カテゴリ用途名
+category-visual-name = カテゴリ表示名
+category-integrations-name = カテゴリ統合名
```
Comment (jared-hughes) [inline localization/ja.ftl:18]: These seem to have translated more than necessary. For example, "カテゴリ統合名" translates back to  "Category integration name", but it should be just "Integrations"

**Review by jared-hughes — COMMENTED** [review]
I know you said
> I do not translate content that, in my judgment, can be understood as is.

But there are *many* more untranslated strings here than in the other language files. 
In particular, most of the "Video Creator" section is untranslated, and many `-name` translations (even if the corresponding `-desc` is translated).

Is it common to mix Japanese and English text on the same page?

File: localization/ja.ftl (hunk 2 above)
Comment (zukizukizuki) [reply localization/ja.ftl]: Thank you for your comment. Fixed.

File: localization/ja.ftl [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+wolfram2desmos-opt-reciprocalExponents2Surds-name = ラジカル記法
+wolfram2desmos-opt-reciprocalExponents2Surds-desc = 1未満の小数の累乗をラジカル等価（surd）に変換する。
+wolfram2desmos-opt-derivativeLoopLimit-name = デリバティブの拡大
+wolfram2desmos-opt-derivativeLoopLimit-desc = ライプニッツ記法のn階微分を反復微分に展開する。（10回まで）
+
+## Pin Expressions
+pin-expressions-name = pinの表現
+pin-expressions-desc = リスト編集モードからのpin式
+pin-expressions-pin = Pin
+pin-expressions-unpin = Unpin
```
[the remark this answers is not in the document]
Comment (zukizukizuki) [reply localization/ja.ftl]: Thank you for your comment. Fixed.

[the remark this answers is not in the document]
Comment (ronwnor) [reply localization/ja.ftl]: 「固定」perhaps? I don't think there's a good verb for "unpin" though. maybe 「外す」.

[the remark this answers is not in the document]
Comment (zukizukizuki) [reply localization/ja.ftl]: It was certainly easier to understand that way. Fixed.

**Comment by zukizukizuki** [comment]
> But there are _many_ more untranslated strings here than in the other language files. In particular, most of the "Video Creator" section is untranslated, and many `-name` translations (even if the corresponding `-desc` is translated).
> 
> Is it common to mix Japanese and English text on the same page?

The parts that I felt did not need translation were the parts where the meaning did not change between Japanese "katakana" and English letters, but it is true that too many English letters are unfriendly, so I modified the translation to "katakana".

**Comment by zukizukizuki** [comment]
> But there are _many_ more untranslated strings here than in the other language files. In particular, most of the "Video Creator" section is untranslated, and many `-name` translations (even if the corresponding `-desc` is translated).
> 
> Is it common to mix Japanese and English text on the same page?

@jared-hughes 
The parts that I felt did not need translation were the parts where the meaning did not change between Japanese "katakana" and English letters, but it is true that too many English letters are unfriendly, so I modified the translation to "katakana".

**Review by jared-hughes — APPROVED** [review]
Thanks for these translations!

**Pull request merged**